tests/acceptance/features/bootstrap/TrashbinContext.php

```diff
@@ -260,16 +260,23 @@ public function theLastWebdavResponseShouldContainFollowingElements(TableNode $e
 		foreach ($elementRows as $expectedElement) {
 			$found = false;
 			$expectedPath = $expectedElement['path'];
-			$expectedMtime = $expectedElement['mtime'] === "deleted_mtime" ? $this->featureContext->getLastUploadDeleteTime() : $expectedElement['mtime'];
+			$expectedMtime = $this->featureContext->getLastUploadDeleteTime();
+			$responseMtime = '';
 			
 			if (isset($expectedElement['mtime'])) {
 				foreach ($files as $file) {
-					if (\ltrim($expectedPath, "/") === \ltrim($file['original-location'], "/") && \ltrim($expectedMtime, "/") === \ltrim($file['mtime'], "/")) {
-						$found = true;
-						break;
+					$responseMtime = $file['mtime'];
+
+					if (\ltrim($expectedPath, "/") === \ltrim($file['original-location'], "/")) {
+						$mtime_difference = \abs((int)\trim($expectedMtime) - (int)\trim($file['mtime']));
+
+						if ($mtime_difference <= 2) {
+							$found = true;
+							break;
+						}
 					}
 				}
-				Assert::assertTrue($found, "$expectedPath with mtime $expectedMtime expected to be listed in response but not found");
+				Assert::assertTrue($found, "$expectedPath expected to be listed in response with mtime $responseMtime but found $expectedMtime");
```

Comment:
```suggestion
				Assert::assertTrue($found, "$expectedPath expected to be listed in response with mtime '$expectedMtime' but found '$responseMtime'");
```

I put single-quotes around the values, because `$responseMtime` might be an empty string, and the single-quotes will make that a bit easier to guess in the error message.